src/app/weather-data/weather-data.component.html

```diff
@@ -1,4 +1,23 @@
 <h1>{{dataLocation.name}}<br>({{dataWithTime.time | date: 'short'}})</h1>
-<p><b>Wind speed: </b> {{dataWithTime.windSpeed10m | windSpeed}}</p>
-<p><b>Chance of rain: </b> {{dataWithTime.probOfPrecipitation | pOPrecip}}</p>
-<p><b>Visibility: </b> {{dataWithTime.visibility | vis}}</p>
+<h2>{{checker}}</h2>
+<p>
+  <b>Wind speed: </b>
```

Comment:
Instead of using bold tags consider styling these with a CSS class

Reply:
I believe i did this because i wanted it all on the same line, maybe just for ease, im not sure. however ive just updated it using span tags with just css styling